editor/src/scene_viewer/mod.rs

```diff
@@ -790,37 +788,54 @@ impl SceneViewer {
                                             .first()
                                             .node(self.scene_gizmo_image)
                                             .screen_position();
-                                    if let Some(action) = self.scene_gizmo.on_click(rel_pos, engine)
-                                    {
-                                        match action {
-                                            SceneGizmoAction::Rotate(rotation) => {
-                                                game_scene.camera_controller.pitch = rotation.pitch;
-                                                game_scene.camera_controller.yaw = rotation.yaw;
-                                            }
-                                            SceneGizmoAction::SwitchProjection => {
-                                                let graph = &engine.scenes[game_scene.scene].graph;
-                                                match graph[game_scene.camera_controller.camera]
-                                                    .as_camera()
-                                                    .projection()
-                                                {
-                                                    Projection::Perspective(_) => {
-                                                        ui.send_message(
-                                                            DropdownListMessage::selection(
-                                                                self.camera_projection,
-                                                                MessageDirection::ToWidget,
-                                                                Some(1),
-                                                            ),
-                                                        );
-                                                    }
-                                                    Projection::Orthographic(_) => {
-                                                        ui.send_message(
-                                                            DropdownListMessage::selection(
-                                                                self.camera_projection,
-                                                                MessageDirection::ToWidget,
-                                                                Some(0),
-                                                            ),
-                                                        );
-                                                    }
+                                    self.scene_gizmo.drag_context = Some(gizmo::DragContext {
+                                        initial_click_pos: rel_pos,
+                                        initial_rotation: gizmo::CameraRotation {
+                                            pitch: game_scene.camera_controller.pitch.to_radians(),
```

Comment:
Why do you need to convert pitch and yaw here to radians? If you look closely at the camera controller code, you'll find that these angles are already in radians.

Reply:
I did check out the camera controller some but when I was logging the positions the numbers reported were not in radians, since I adjusted the rotation messaging this may not be happening anymore. Ill test it out and let you know.

Reply:
Yep removing radians works doesn't affect functionality now that the messaging is set up properly.